src/main/scala/io/github/pauljamescleary/petstore/infrastructure/endpoint/alias.scala

```diff
@@ -0,0 +1,11 @@
+package io.github.pauljamescleary.petstore
+package infrastructure.endpoint
+
+import io.github.pauljamescleary.petstore.domain.users.User
+import org.http4s.Response
+import tsec.authentication.{AugmentedJWT, SecuredRequest, TSecAuthService}
+
+package object alias {
```

Comment:
why not make this `package object endpoint`, and then you wouldn't have to `import alias._` within the endpoint package.

Reply:
How sure you are about that ? I tried renaming alias for endpoint and you still need to import endpoint._ for the aliases.